src/Discord.Net/API/Gateway/IdentifyParams.cs

```diff
@@ -14,5 +14,7 @@ public class IdentifyParams
         public int LargeThreshold { get; set; }
         [JsonProperty("compress")]
         public bool UseCompression { get; set; }
+        [JsonProperty("shard", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
We avoid using Null/Default ValueHandling - use the Optional<T> type instead which on serializes if it was assigned a value.